lib/globals.js

```diff
@@ -55,13 +60,7 @@ async function launchDynamoDbLocalDocker() {
   );
 }
 
-async function teardownDynamoDbLocalDocker(jestConfig = {}) {
-  const containerId = global[Symbol.for('ddbLocalContainerId')];
-  if (containerId && !jestConfig.watch && !jestConfig.watchAll) {
-    await exec(`docker kill ${containerId}`);
-  }
-  return;
-}
+function teardownDynamoDbLocalDocker() {}
```

Comment:
@krewllobster Do we need to keep this empty function around for some reason?

Reply:
Nope.